<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 76, 70)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>541</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 194, 53)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18pt</string>
     </property>
     <property name="text">
      <string>𝑯𝒐𝒘 𝑫𝒐 𝒀𝒐𝒖 𝑾𝒂𝒏𝒕 𝑻𝒐 𝑷𝒂𝒚?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CustomerLogin">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 66, 66); font-size: 15pt</string>
     </property>
     <property name="text">
      <string>Online</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CustomerLogin_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 66, 66); font-size: 15pt</string>
     </property>
     <property name="text">
      <string>Cash On Delivery</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CustomerLogin_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>400</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 66, 66); font-size: 15pt</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
